src/drivers/net/ena.h

```diff
@@ -506,6 +514,22 @@ struct ena_acq {
 	unsigned int phase;
 };
 
+struct ena_aenq_event {
+	uint16_t group;
+	uint16_t syndrom;
+	uint8_t flags;
+	uint8_t reserved[3];
+	uint32_t timestamp_low;
+	uint32_t timestamp_high;
+	uint32_t data[12];
+};
+
+/** Async event notification queue */
+struct ena_aenq {
+	/** Notifications */
+	struct ena_aenq_event *events;
```

Comment:
Since the array fields within the other structures are `req` and `rsp`, both of which are abbreviated and neither of which are pluralised, `evt` would seem more consistent than `events`.